greenpeace/planet4-master-theme
# PLANET-6964 Remove background image on listing pages by default

Ref: https://jira.greenpeace.org/browse/PLANET-6964

---

This removes the feature flag and removes the background image and overlay from being used in listing pages.

I left the tag image functionality intact because the image that admins add there can be used in opengraph metadata of the listing pages. Currently it's not being used, but I'll open a follow up PR for that.

### Testing

1. Go to Post > Tags and create a new tag.
2. Add title, description, image but don't pick a redirect page.
3. Check the tag on the frontend. Its background should be just the normal background color instead of the image being selected in the previous step.
